package-lock.json

```diff
@@ -1,8 +1,3429 @@
 {
```

Comment:
I see package-lock.json changing a lot between PRs.
Can we add it to .gitignore if it's not being used?